clients/lsp-pyls.el

```diff
@@ -481,10 +481,13 @@ So it will rename only references it can find."
  (make-lsp-client :new-connection (lsp-stdio-connection
                                    (lambda () lsp-clients-python-command))
                   :major-modes '(python-mode cython-mode)
-                  :priority -1
+                  :priority -2
                   :server-id 'pyls
                   :library-folders-fn (lambda (_workspace) lsp-clients-python-library-directories)
                   :initialized-fn (lambda (workspace)
+                                    (warn (concat "The palantir python-language-server (pyls) is unmaintained; "
```

Comment:
Can you add one defcustom something like `lsp-pyls-disable-warning` and do something like
```
(unless lsp-pyls-disable-warning
   (warn ...
```
and mention `lsp-pyls-disable-warning` in the warn?

Reply:
Sure, this is added